packages/tabsheet/test/visual/lumo/tabsheet.test.js

```diff
@@ -60,6 +60,38 @@ describe('tabsheet', () => {
           await visualDiff(div, `${dir}-bordered`);
         });
       });
+
+      describe('theme', () => {
+        beforeEach(() => {
+          element = fixtureSync(
+            `
+            <vaadin-tabsheet>
+            <div slot="prefix">Prefix</div>
+            <div slot="suffix">Suffix</div>
+            
+            <vaadin-tabs slot="tabs">
+            <vaadin-tab id="tab-1">Tab 1</vaadin-tab>
+            </vaadin-tabs>
+            
+            <div tab="tab-1">
+            Odio quis mi. Aliquam massa pede, pharetra quis, tincidunt quis, fringilla at, mauris. Vestibulum a massa.
+            Vestibulum luctus odio ut quam. Maecenas congue convallis diam. Cras urna arcu, vestibulum vitae, blandit ut,
+            laoreet id, risus. Ut condimentum, arcu sit amet placerat blandit, augue nibh pretium nunc, in tempus sem dolor
+            non leo. Etiam fringilla mauris a odio. Nunc lorem diam, interdum eget, lacinia in, scelerisque sit amet, purus.
+            Nam ornare. Donec placerat dui ut orci. Phasellus quis lacus at nisl elementum cursus. Cras bibendum egestas
+            nulla. Phasellus pulvinar ullamcorper odio. Etiam ipsum. Proin tincidunt. Aliquam aliquet.
+            </div>
+            </vaadin-tabsheet>
+            `,
+            div,
+          );
+        });
+
+        it('no-padding', async () => {
```

Comment:
I don't think you need another `describe` here. Consider placing the test next to `bordered` one.

Reply:
I was actually thinking of moving the bordered one into that `describe` later to have all theme related tests in one suite, but I can take it out if you don't think that's necessary.

Reply:
We don't have many tests here for now so I think single `describe` is fine.

Reply:
Sure!